specification/communication/data-plane/Microsoft.CommunicationServicesSms/stable/2021-03-07/communicationservicessms.json

```diff
@@ -0,0 +1,209 @@
+            "description": "Represents the body of the send message request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SendMessageRequest"
+            }
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "description": "Version of API to invoke.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/SmsSendResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SmsRecipient": {
+      "description": "Recipient details for sending SMS messages.",
+      "required": [
+        "to"
+      ],
+      "type": "object",
+      "properties": {
+        "to": {
+          "description": "The recipient's phone number in E.164 format.",
+          "type": "string"
+        },
+        "repeatabilityRequestId": {
+          "description": "If specified, the client directs that the request is repeatable; that is, the client can make the request multiple times with the same Repeatability-Request-ID and get back an appropriate response without the server executing the request multiple times. The value of the Repeatability-Request-ID is an opaque string representing a client-generated, 36-character hexadecimal case-insensitive encoding of a UUID (GUID), identifier for the request.",
+          "type": "string"
+        },
+        "repeatabilityFirstSent": {
+          "description": "MUST be sent by clients to specify that a request is repeatable. Repeatability-First-Sent is used to specify the date and time at which the request was first created.eg- Tue, 26 Mar 2019 16:06:51 GMT",
+          "type": "string"
+        }
+      }
+    },
+    "SmsSendOptions": {
+      "description": "Optional configuration for sending SMS messages.",
+      "required": [
+        "enableDeliveryReport"
+      ],
+      "type": "object",
+      "properties": {
+        "enableDeliveryReport": {
+          "description": "Enable this flag to receive a delivery report for this message on the Azure Resource EventGrid.",
+          "type": "boolean",
+          "default": false
+        },
+        "tag": {
+          "description": "Use this field to provide metadata that will then be sent back in the corresponding Delivery Report.",
+          "type": "string"
+        }
+      }
+    },
+    "SendMessageRequest": {
+      "description": "Represents the properties of a send message request.",
+      "required": [
+        "message",
+        "smsRecipient",
+        "from"
+      ],
+      "type": "object",
+      "properties": {
+        "from": {
+          "description": "The sender's phone number in E.164 format that is owned by the authenticated account.",
+          "type": "string"
+        },
+        "smsRecipient": {
```

Comment:
since this now expects an array of values, would it be more intuitive to rename it smsRecipients?

Reply:
Done